src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -882,6 +885,20 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
         }
     }
 
+    private stateEventsContainOngoingLegacySession(callMemberEvents: Map<string, MatrixEvent>): boolean {
+        for (const callMemberEvent of callMemberEvents.values()) {
+            const content = callMemberEvent.getContent();
+            if (Array.isArray(content["memberships"])) {
+                for (const membership of content.memberships) {
+                    if (!new CallMembership(callMemberEvent, membership).isExpired()) {
```

Comment:
I am currently not sure if : `!new Obj().is_sth()` works or if it is preferred to do: `!(new Obj().is_sth())`

Reply:
I wasn't sure either, but the linter forces the parentheses to go away, so it must be fine.